src/htmlTransformers.js

```diff
@@ -60,18 +60,20 @@ export const transformAsides = content => {
   });
 };
 
-export const transformRelatedContent = (content, lang) => {
+export const transformRelatedContent = (content, lang, options) => {
   content('div').each((_, div) => {
     const isRelatedContentGroup =
       div.attribs && div.attribs['data-type'] === 'related-content';
     if (isRelatedContentGroup) {
-      const relatedArticleList = createRelatedArticleList(
-        { locale: lang, articleCount: content(div).children().length },
-        content(div)
-          .children()
-          .toString()
-      );
-      content(div).before(relatedArticleList);
+      if (!options.removeRelatedContent) {
```

Comment:
You can probably remove this check now?

Reply:
createRelatedArticleList must not run, so I still need to check if relatedContent should be removed or not.

Reply:
Should'nt be true if you move the `options.removeRelatedContent` check to the embedToHtml function in relatedContentPlugin. We don't want to send `embed` tags back to the consumer.

Reply:
Well, the div with `data-type=related-content` is still present even if I return a empty string from embedsToHtml function, and therefore, the related article header will show.